wpilibj/src/shared/java/edu/wpi/first/wpilibj/command/CommandGroup.java

```diff
@@ -34,14 +34,14 @@
   /**
    * The commands in this group (stored in entries).
    */
-  private final Vector m_commands = new Vector();
+  private final List<Entry> m_commands = new ArrayList<>();
```

Comment:
I just looked through the codebase. We only create new threads for the PID update functions. Everything else runs in the same thread.